module/hotspot/hsperfdata/hsperfdata.go

```diff
@@ -89,59 +162,67 @@ func (m *MetricSet) buildMapStr(entries []PerfDataEntry) common.MapStr {
 	return event
 }
 
-func (m *MetricSet) publishAll() (common.MapStr, error) {
-	f, err := os.Open(m.hsPerfDataPath)
+func (p *ProcStats) publishAll() (common.MapStr, error) {
+	f, err := os.Open(p.hsPerfDataPath)
 	if err != nil {
 		return nil, err
```

Comment:
We can log HSBeat can access `hsperfdata` as below:

`hsperfdata.go`
```
    165 func (p *ProcStats) publishAll() (common.MapStr, error) {
    166         f, err := os.Open(p.hsPerfDataPath)
    167         if err != nil {
    168  +              logp.Debug(DEBUG_SELECTOR, "Could not open %v (%v)", p.h        sPerfDataPath, err)
    169                 return nil, err
    170         }
```

debug message:
```
2016/12/31 14:14:27.726070 hsperfdata.go:120: DBG  Found 1 running java processes
2016/12/31 14:14:27.727269 hsperfdata.go:168: DBG  Could not open /tmp/hsperfdata_ysuenaga/1641 (open /tmp/hsperfdata_ysuenaga/1641: permission denied)
```